pages/docs/reference/using-gradle.md

```diff
@@ -154,49 +175,97 @@ With Gradle Kotlin DSL, configure source sets with `java.sourceSets { ... }` ins
 
 When targeting JavaScript, a different plugin should be applied:
 
-<div class="sample" markdown="1" mode="groovy" theme="idea">
+<div class="multi-language-sample" data-lang="groovy">
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-highlight-only>
 
 ``` groovy
-apply plugin: "kotlin2js"
+plugins {
+    id 'kotlin2js' version '{{ site.data.releases.latest.version }}'
+}
 ```
 
+</div>
 </div>
 
-This plugin only works for Kotlin files so it is recommended to keep Kotlin and Java files separate (if it's the case that the same project contains Java files). As with
-targeting the JVM, if not using the default convention, we need to specify the source folder using *sourceSets*:
+<div class="multi-language-sample" data-lang="kotlin">
+<div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
 
-<div class="sample" markdown="1" mode="groovy" theme="idea">
+```kotlin
+plugins {
+    kotlin("js") version "{{ site.data.releases.latest.version }}"
```

Comment:
This won't work either, for the same reason as the Groovy sample above. As far as I can see, our New Project wizard adds the workaround to the `settings.gradle` file and applies the plugin as `id("kotlin2js")` in Kotlin DSL, too.

Reply:
+